U/USRPHardwareDriver/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "USRPHardwareDriver"
+version = v"4.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/EttusResearch/uhd.git", "f633b497ff931c3c180d00148fc66e4feb8b8b6a")
+]
+
+dependencies = [
+    Dependency("boost_jll"; compat="=1.76.0"), # max gcc7
+    Dependency("libusb_jll", compat="1.0.24"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd uhd/host
+
+#code generators, only needed at build time
+apk add py3-mako py3-ruamel.yaml
+
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DENABLE_EXAMPLES=OFF \
+    -DENABLE_TESTS=OFF \
+    -DENABLE_PYTHON_API=OFF \
+    ..
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+# TODO: Windows has several issues with boost threads. There is a WIP branch:
+# https://github.com/JuliaTelecom/uhd/tree/juliatelecom/patch-v4.1.0.1
+platforms = filter!(p -> !Sys.iswindows(p) && !in(arch(p),("armv7l","armv6l")), supported_platforms(;experimental=true))
+platforms = expand_cxxstring_abis(platforms)
+# For some reasons, building for CXX03 string ABI doesn't actually work, skip it
```

Comment:
This is potentially problematic. What is the error?

Reply:
I added another commit removing it. I get the warning typically with Boost telling me to add it: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=13355&view=results 

Though it looks like it expanded to `cxx11` automatically?

Reply:
I built locally, and I can expand to both `cxx03` and `cxx11` no problem, so I pushed a commit here to test in on Yggdrasil.